website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -107,16 +109,51 @@ The expected use case is to create admin partitions on Kubernetes clusters. This
 
 The following procedure will result in an admin partition in each Kubernetes cluster. The Consul clients running in the cluster with servers will be in the `default` partition. Another partition called `clients` will also be created.
 
+#### Prepartion for install across multiple Kubernetes clusters
+
 Verify that your Consul deployment meets the [Kubernetes Requirements](#kubernetes-requirements) before proceeding.
 
 1. Verify that your VPC is configured to enable connectivity between the pods running Consul clients and servers. Refer to your virtual cloud provider's documentation for instructions on configuring network connectivity.
-1. Create the license secret in each cluster, e.g.:
+1. Set environment variables to use with shell commands. 
+
+  ```shell-session
+  $ export HELM_RELEASE_SERVER=server
+  $ export HELM_RELEASE_CLIENT=client
+  $ export SERVER_CONTEXT=<context for server, run `kubectl config current-context` for cluster provisioned for servers>
+  $ export CLIENT_CONTEXT=<context for workload partition, run `kubectl config current-context` for cluster provisioned for workload partition>
+  ```
+1. Set your context to the server cluster. 
+
+  ```shell-session
+  $ kubectl config use-context ${SERVER_CONTEXT}
+  ```
+  
+1. Create the license secret in server cluster.
 
   ```shell-session
-  $ kubectl create secret generic license --from-file=key=[license file path i.e. ./license.hclic]
+  $ kubectl create secret -n consul generic license --from-file=key=./path/to/license.hclic
```

Comment:
```suggestion
  $ kubectl create secret --namespace consul generic license --from-file=key=./path/to/license.hclic
```
I think we want to always use the expanded form of the flags. Also, is `consul generic license` an actual value or is it placeholder text? We've been trying to use brackets (square or angle) for placeholder text